src/components/Calculator.js

```diff
@@ -0,0 +1,23 @@
+function Calculator() {
+  const keys = ['AC', '+/-', '%', '÷', 7, 8, 9, 'x', 4, 5, 6, '-', 1, 2, 3, '+', 0, '.', '='];
+
+  return (
+    <section id="calculator">
+      <input
+        type="text"
+        value="0"
+        disabled
+        id="exit"
+      />
+      <section id="input">
+        <article id="digits">
+          {
+                        keys.map((but) => <button key={but} type="button">{but}</button>)
+                    }
+        </article>
+      </section>
+    </section>
+  );
+}
```

Comment:
- [ ] Please reduce the with of each calculator key and increase the height / just add padding-top & bottom to the keys so that they match the design. 

**Your implementation**
![desgn](https://github.com/kazaneza/math-magician/assets/99656687/02e3a7aa-3bb7-4a63-a29f-077ee666fd68)

**required design**
![des](https://github.com/kazaneza/math-magician/assets/99656687/a3b2be60-cdcc-4406-b3e2-24dc0f536355)